docs/source/technical_tutorials/user_data/memory-profiling-python.ipynb

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
<p>My suggestion would be to shorten these headers some. Maybe something like "Step 1: Installation" and then in normal text below, put "Install <code>memory-profiling</code> in your existing working environment" and any additional details as necessary (I see Jamison's comment below on more elaboration for Step 3). I'd probably update that format for steps 1-4, I think the formatting for 5 is okay besides also updating the capitalization. Overall, I think it looks good though!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks, @smk0033. I've added description and shortened the headings. </p>